compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/insert_loops_for_vectorization.mlir

```diff
@@ -257,3 +267,84 @@ module {
     return %0 : tensor<4x6x8xbf16>
   }
 }
+
+// -----
+
+// CHECK-LABEL: @element_wise_bufferized
```

Comment:
What are you specifically checking for here that wasn't covered earlier?

Reply:
Collapsing of unit dims as well as `tensor` vs `memref` args.